client/structure/login.js

```diff
@@ -40,7 +40,13 @@ export function LoginScreen({ action }) {
         const personaPreview = await getFirebaseDataAsync([
             'silo', datastore.getSiloKey(), 'structure', 'profile',
             'instance', personaKey, 'global', 'preview']);
+        const fbUser = datastore.getFirebaseUser();            
         if (!personaPreview?.name) {
+            const isCelebrity = await datastore.callServerAsync(
+                'verify', 'getIsCelebrity', {name: fbUser.displayName});
+            if (isCelebrity) {
+                await datastore.callServerAsync('verify', 'setVerficationStatusOnPersona', {email: fbUser.email})
```

Comment:
Email passed from the client isn't secure. Better for the server side to grab the email of the account.

Reply:
done